### PR TITLE
Mystrom binding, review http calls to Mystrom server

### PR DESCRIPTION
**Pull request by coolweb**
MyStrom complains about the number of calls on the server and asked me
to review the way I use their API. So I change the http calls and all is
tested and validated.

### CODE REVIEW

File: bundles/binding/org.openhab.binding.mystromecopower/pom.xml [hunk 1]
```diff
@@ -32,4 +32,5 @@
 		</plugins>
 	</build>
 
+	<version>1.8.1</version>
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.mystromecopower/pom.xml]: remove version tag

File: bundles/binding/org.openhab.binding.mystromecopower/pom.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>org.openhab.bundles</groupId>
 		<artifactId>binding</artifactId>
-		<version>1.7.0-SNAPSHOT-SNAPSHOT</version>
+		<version>1.8.0-SNAPSHOT-SNAPSHOT</version>
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.mystromecopower/pom.xml]: change to `1.9.0-SNAPSHOT`

File: bundles/binding/org.openhab.binding.mystromecopower/src/main/java/org/openhab/binding/mystromecopower/internal/api/mock/MockMystromClient.java [hunk 3]
```diff
@@ -82,4 +82,10 @@ public Boolean ChangeState(String deviceId, Boolean newStateIsOn) {
 	public void RestartMaster(String deviceId) {
 	}
 
+	@Override
+	public List<MystromDevice> getDevicesState() {
+		// TODO Auto-generated method stub
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.mystromecopower/src/main/java/org/openhab/binding/mystromecopower/internal/api/mock/MockMystromClient.java]: remove comment

**Comment by teichsta** [comment]
Thanks @coolweb for this contribution. Please find my review comments inline. Could you please also resolve the conflicts and squash your commits into one, after all?

**Comment by teichsta** [comment]
Any updates @coolweb?

**Comment by coolweb** [comment]
Sorry I'm very busy for the moment for my work, I don't forget this pull
request...

**Comment by teichsta** [comment]
> Sorry I'm very busy for the moment for my work, I don't forget this pull
request...

**Comment by teichsta** [comment]
unfortunately your changes doesn't seem to respect the current code formatting. The diff is quite unreadable hence. Could you please try to fix the formatting and squash your commits into one after all? 

Note: PRs are best send from feature branches not your current master.

**Comment by coolweb** [comment]
@teichsta  ok, how could I do a feature branch? Do I need to recreate a new openhab eclipse dev env to do my commit? Thx.

**Comment by coolweb** [comment]
@teichsta I squashed my 2 last commits. Could you reviewed them?
Thx.

**Comment by teichsta** [comment]
@coolweb any chance to revive this branch/PR again?

**Comment by coolweb** [comment]
@teichsta  This PR was closed due to branch problem on my PC, so a new PR was created and already accepted: #4295

**Pull request closed without merging**